src/main/resources/jenkins/scm/impl/trait/WildcardSCMSourceFilterTrait/help-excludes.html

```diff
@@ -22,6 +22,6 @@
  ~ THE SOFTWARE.
  -->
 <div>
-    Project name patterns to ignore even if matched by the includes list.
-    For example: <code>release</code>
+    Project name patterns (separated by space) to ignore even if matched by the includes list.
+    For example: <code>release alpha-* beta-*</code>
```

Comment:
Copy-pasted example, was misleading. Presumably these should look like names of projects, rather than names of branches.

Reply:
Yeah, good point. I see this pattern used in includes help file as well, fixed.